plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/analyses/BranchWalker.java

```diff
@@ -20,28 +21,37 @@
 import org.eclipse.n4js.flowgraphs.model.RepresentingNode;
 import org.eclipse.n4js.n4JS.ControlFlowElement;
 
-/** see {@link PathWalkerInternal} */
-abstract public class PathWalker extends PathWalkerInternal {
-	ControlFlowElement pLastCFE;
+/** see {@link BranchWalkerInternal} */
+abstract public class BranchWalker extends BranchWalkerInternal {
```

Comment:
Regarding the rename `PathWalker` -> `BranchWalker` (and the corresponding renames of other classes and newly added classes): did you update the design doc?

What about other adjustments of the explanations in the design doc?

Reply:
Yes, I also updated the design doc (see other PR).